vignettes/articles/04-read-the-outputs.Rmd

```diff
@@ -40,4 +40,218 @@ analysis.
+all companies with production in the given sector. Value in percentage points.
+* `sector_value_change`: Potential absolute value change of the portfolio or
+loan book on the sector level indicated. Calculated as
+`sector_exposure * VaR_sector`.
+* `analysed_sectors_exposure`: Exposure of the portfolio or loan book to all
+sectors in scope of the analysis. This is an aggregate of the exposure across
+all companies with production in the sectors that are in scope.
+* `VaR_analysed_sectors`: Relative potential value change of portfolio or loan
+book across all companies with production in the in-scope sectors. Value in
+percentage points.
+* `analysed_sectors_value_change`: Potential absolute value change of the
+portfolio or loan book on the level of in scope sectors. Calculated as
+`analysed_sectors_exposure * VaR_analysed_sectors`.
+* `portfolio_aum`: Total absolute value of the assets under management for the
+given asset type.
+* `portfolio_value_change_perc`: Relative potential value change of portfolio or
+loan book across all companies relative to the total portfolio value. Note that
+this does not cover potential losses for out of scope companies. Value in
+percentage points.
+* `portfolio_value_change`: Potential absolute value change of the portfolio or
+loan book. Calculated as `portfolio_aum * portfolio_value_change_perc`.
+* `exclude`: Logical indicator that shows if the given combination of company
+and technology was excluded from the shock because the production forecast
+indicated a phase out in production.
+
+### stress_test_results_loans_port_\<suffix\>
+
+This result file contains relative expected value changes on the portfolio level,
+based on the DCF model to calculate market risk. It additionally contains
+aggregations on the technology, sector and total levels. The columns and rows in
+this file are a subset of the ones included in the company level output.
+The user can refer to the explanations above for reference.
+
+### stress_test_results_loans_comp_el_\<suffix\>
+
+This result file contains expected changes in the probability of default and in
+expected losses. The company level pd changes are used to derive loan book level
+changes in expected loss.
+
+* `scenario_name`: Name that highlights the use of the Carbon budget.
+compensation mechanism and the Year of the introduction of a transition policy.
+* `scenario_geography`: Region for which the analysis was run. This refers to
+the location of assets considered as well as the regionality of scenario targets
+used.
+* `investor_name`: Name of the investor or bank for which the analysis was run.
+* `portfolio_name`: Name of the portfolio or loan book managed by the investor
+or bank for which the analysis was run.
+* `company_name`: Name of the company that was analyzed.
+* `ald_sector`: Sector of the production assets analyzed.
+* `pd`: Starting point probability of default for loans to the given company in
+the loan book in percentage points.
```

Comment:
no. we are in the weird situation that we try to tailor this version specifically to banks which will solely run it on loan books. This is why some of the formulations are biased to that. At the same time, I think it would be a bad idea to write separate vignettes for all asset types because of the big amount of duplication there. Not quite sure how to best handle this..